command/server/config.go

```diff
@@ -60,8 +60,10 @@ func DevConfig(ha, transactional bool) *Config {
 			&Listener{
 				Type: "tcp",
 				Config: map[string]interface{}{
-					"address":     "127.0.0.1:8200",
-					"tls_disable": "1",
+					"address":                      "127.0.0.1:8200",
+					"tls_disable":                  true,
+					"proxy_protocol":               "use_if_authorized",
```

Comment:
I wonder if a nested `proxy_protocol` block would be nicer here, with keys of `behavior` and `allowed_addrs`.